src/sources/sirius/persons.njk

```diff
@@ -0,0 +1,51 @@
+---
+title: Persons Table - Sirius Case Management
+layout: layouts/fullwidth.njk
+---
+<h1 class="govuk-heading-l">{{title}}</h1>
+<p class="govuk-body-l"> The persons table includes data on people within the Sirius system. It contains both LPA, Guardianship and Supervision data, as well as data relating to persons who are informed parties.</p>
+<p class="govuk-body"><strong>Stability:</strong> The persons table is to be considered <strong>unstable</strong> and likely to change due to ongoing development work.</p>
+
+<table class="govuk-table govuk-!-font-size-16">
```

Comment:
Can you use the nunjucks macro here https://github.com/ministryofjustice/opg-modernising-lpa-docs/blob/fae8b8bd5dfc34cb7e581376db47b3f80d0d1480/src/adr/index.njk or does it not give you the flexibility?

Reply:
I'll check! I've forced the font-size down to allow for the data and want to add some scaffolding for pinned headers, so as long as i can send in a few custom classes/data attributes it should be fine.

Reply:
Looks like no data attributes in the macro, also as I'm composing two json files together I'd have to rework that too. Might be one for a later PR.